modules/retail/pages/retail-upgrade-branchserver.adoc

```diff
@@ -4,4 +4,15 @@
 This section describes upgrading a {smr} Branch Server to the next SP (service pack).
 
 {smr} Branch Server is a client system such as a {productname} Proxy with additional {smr} features.
-For more information about upgrading a proxy client, see xref:upgrade:proxy-update.adoc[].
+
+[NOTE]
+====
+Upgrade the {productname} Server before beginning with the {srm} upgrade.
+====
+
+
```

Comment:
```suggestion
```

Reply:
I think we want 3 empty lines before a procedure title ;)